arduino/smartcar/smartcar.ino

```diff
@@ -46,16 +50,37 @@ const auto DEFAULT_DRIVING_SPEED = 1.5;
 void setup()
 {
     Serial.begin(9600);
-
+	mqtt.begin("hysm.dev", 1883, WiFi);
+	if (mqtt.connect("arduino", "", "")) {
+		mqtt.subscribe("/smartcar/control/#", 1);
+		mqtt.onMessage([](String topic, String message) {
+			Serial.println(topic + " " + message);
+			
+			mqtt.publish("/smartcar/received/msg", message);
+			
+			if (topic == "/smartcar/control/speed") {
+				car.setSpeed(message.toInt());
+			} else if (topic == "/smartcar/control/steering") {
+				car.setAngle(message.toInt());
```

Comment:
Some sort of value sanitization needs to be done here. It's obvious that we're expecting an int to be sent to both topics, but we're never ensuring that the value we receive will be an int. If I were to send a message along the lines of "abc" to one of these topics, the best-case scenario is that we will get an exception.